docs/testing/application-specific-api-error-handling.md

```diff
@@ -64,7 +61,7 @@ The following table is a list of errors that the API may return.
 |`AccessDenied` |You cannot perform the requested operation.| |
 |`ActivityLimitReached`|Activity limit has been reached.| |
 |`ApiNotAvailable`|The requested API is not available.| |
-|`ApiNotFound`|The API you are trying to use could not be found. It may be available in a newer version of Excel. See the [Excel JavaScript API requirement sets](/javascript/api/requirement-sets/excel/excel-api-requirement-sets) article for more information.| |
+|`ApiNotFound`|The API you are trying to use could not be found. It may be available in a newer version of Excel. See the [Excel JavaScript API requirement sets](/javascript/api/requirement-sets) article for more information.| |
```

Comment:
Why change the URL but not the link text?

Reply:
I didn't fully undo this change...
I wanted to change what page this sends people to, but realized the error text will be in the product.